### PR TITLE
Make durationInFrames optional

### LINKED ISSUE #526: Proposal: Allow to omit `durationInFrames={Infinity}` for sequence
[Issuehunt badges](https://issuehunt.io/r/JonnyBurger/remotion/issues/526)

We can make it that if you omit it this is assumed as default. Makes the syntax a bit more concise.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $112.00)

- [jonnyburger jonnyburger](https://issuehunt.io/u/jonnyburger) ($112.00)

### Submitted pull Requests
- [#621 Make durationInFrames optional](https://issuehunt.io/r/JonnyBurger/remotion/pull/621)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/JonnyBurger/remotion/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by khalid283**
Resolve #526 
Screenshot 2021-10-02 at 4 16 56 PM

There are two references of `durationInFrames={Infinity}` in the test that I have not removed, letting you decide.

Check locally by making changes in the example project and run docs to check doc changes. Also, eslint rules.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#526: Proposal: Allow to omit `durationInFrames={Infinity}` for sequence](https://issuehunt.io/repos/274495425/issues/526)
---
</details>

### CODE REVIEW

**Comment by JonnyBurger** [comment]
It is fantastic! 👏 

I did some testing and it worked super well 😎 ESLint rule is super cool too! Thanks a lot 🥳

**Review by JonnyBurger — APPROVED** [review]

**Comment by khalid283** [comment]
Will make other PR in the template repo.

**Comment by JonnyBurger** [comment]
That would be fantastic @khalid283!
I am in the process of merging and releasing.

**Review by JonnyBurger — APPROVED** [review]

File: packages/eslint-plugin/src/tests/no-duration-frames-infinity.test.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    `,
+  ],
+  invalid: [
+    {
+      code: `
+        import {Composition} from 'remotion';
+
+        export const Re = () => {
+          return (
+            <Sequence>
```
Comment (khalid283) [inline packages/eslint-plugin/src/tests/no-duration-frames-infinity.test.ts]: @JonnyBurger this has to be invalid when `durationInFrames={Infinity}`. That's reason build failed, I will push to update this.

Comment (khalid283) [reply packages/eslint-plugin/src/tests/no-duration-frames-infinity.test.ts]: You fixed it closing

Comment (JonnyBurger) [reply packages/eslint-plugin/src/tests/no-duration-frames-infinity.test.ts]: Yep I fixed it! Now the ESLint rule will even suggest a fix 😎

**Comment by JonnyBurger** [comment]
All templates are up to date and a new version was released, thanks a lot for your contribution @khalid283! 🎉

**Pull request merged**